Eventonica/client/src/components/Users.jsx

```diff
@@ -0,0 +1,79 @@
+import { useState } from "react";
+import DeleteUser from "./components/DeleteUser";
```

Comment:
Hint as to why you are having errors: is this the correct path to DeleteUser.jsx?

Reply:
Fixed it ! "./DeleteUser" since they are both in the components folder it isn't needed to access it